### PR TITLE
feat: add Back navigation

### LINKED ISSUE #474: Back navigation
We should implement a back navigation for our pages.
This should be a small arrow representing "Go Back" <-

For example we have the pages Accounts->Payment Pointers->Transactions, and maybe a breadcrumb navigation would help here the users. We can also think on having them on all pages and the back button bringing us back to home page. This would be especially helpful for mobile users, where the main menu is not visible all the time.

The back <= should appear on the very top left side.

### PR DESCRIPTION
**Pull request by umeshSinghVerma**
## Issue
Fixes #474 

## Description

Hello @Tymmmy  I've made some updates to the Interledger Testnet and Logo Duo. I added a link to the home page for better navigation.

In terms of navigation, I've decided not to include a back button in the desktop view, as I believe it's not necessary. However, I've added a back button specifically for the mobile view to enhance user experience.

Regarding account creation in the local development environment, I'm currently facing some issues, so I can't access certain routes. However, I suspect that adding a back button in front of the balance might help resolve this problem. I'd appreciate it if you could take a look and let me know if any further modifications are needed.

I've attached screenshots for your reference. Please review and provide your feedback.
Screenshot from 2023-10-07 14-42-44
Screenshot from 2023-10-07 14-43-22
Screenshot from 2023-10-07 14-43-07

Thanks!

### CODE REVIEW

**Comment by Tymmmy** [comment]
In order to create an account you need some local assets added into rafiki-admin.
http://localhost:3012
For example add an asset of :
image
Then you will be able to create an account for USD.

File: packages/wallet/frontend/src/components/PageHeader.tsx [hunk 1]
```diff
@@ -1,15 +1,25 @@
...
+import Backbutton from './icons/Backbutton'
+import { usePathname } from 'next/navigation'
 type PageHeaderProps = {
   title: string
   message?: ReactNode
 }
 
 export const PageHeader = ({ title, message }: PageHeaderProps) => {
+  const pathname = usePathname()
+  console.log('this is the path name', pathname)
```
Comment (Tymmmy) [inline packages/wallet/frontend/src/components/PageHeader.tsx:11]: remove all console logs

File: packages/wallet/frontend/src/components/icons/Backbutton.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { useRouter } from 'next/router'
+export default function Backbutton() {
+  const router = useRouter()
+  return (
+    <button
+      className="flex-start mr-5 mt-[5px] flex h-4 w-4 scale-125 items-center active:scale-100 md:hidden"
+      onClick={() => {
+        // router.push(location)
```
Comment (Tymmmy) [inline packages/wallet/frontend/src/components/icons/Backbutton.tsx:9]: remove commented out code

File: packages/wallet/frontend/src/components/icons/Backbutton.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        xmlnsXlink="http://www.w3.org/1999/xlink"
+        version="1.1"
+        x="0px"
+        y="0px"
+        viewBox="0 0 78.415 98.02"
+        enableBackground="new 0 0 78.415 78.416"
+        xmlSpace="preserve"
+      >
+        <g>
+          <g>
```
Comment (Tymmmy) [inline packages/wallet/frontend/src/components/icons/Backbutton.tsx:24]: remove one level of <g>

**Comment by Tymmmy** [comment]
When we are on Send or Request pages, the Total balance text has moved to the left:
image

If you check rafiki.money you see that the text was in the center:
image


In the account details page the arrow is a bit not centered to Balance text:
image


On Safari the button does not show:
image

**Comment by umeshSinghVerma** [comment]
Thanks @Tymmmy I will try to solve all these issues as soon as possible also I did not understand what do you mean by  -- remove one level of

File: packages/wallet/frontend/src/components/icons/Backbutton.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
```
Comment (Tymmmy) [inline packages/wallet/frontend/src/components/icons/Backbutton.tsx:23]:
```suggestion
```

File: packages/wallet/frontend/src/components/icons/Backbutton.tsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+            <path
+              fill="#003A2F"
+              d="M0,39.208c0,21.654,17.554,39.208,39.208,39.208c21.653,0,39.207-17.554,39.207-39.208S60.861,0,39.208,0    C17.554,0,0,17.554,0,39.208z M24.924,36.816l18.511-18.512c0.66-0.66,1.525-0.99,2.391-0.99s1.731,0.33,2.391,0.99    c1.316,1.319,1.32,3.458,0,4.777L32.088,39.209l16.128,16.125c1.316,1.32,1.316,3.459,0,4.777c-1.32,1.32-3.461,1.32-4.775,0    L24.924,41.598C23.604,40.278,23.604,38.136,24.924,36.816z"
+            />
+          </g>
```
Comment (Tymmmy) [inline packages/wallet/frontend/src/components/icons/Backbutton.tsx:28]:
```suggestion
```

Comment (umeshSinghVerma) [reply packages/wallet/frontend/src/components/icons/Backbutton.tsx:28]: Thanks :) got it

**Comment by umeshSinghVerma** [comment]
Hello @Tymmmy I have tried to fix all the bugs you have mentioned, kindly check them once.

File: packages/wallet/frontend/src/components/icons/Backbutton.tsx
Comment (Tymmmy) [inline packages/wallet/frontend/src/components/icons/Backbutton.tsx:1]: Please move this file into components folder, as it is a component, and not really just an icon.

**Comment by Tymmmy** [comment]
The functionality looks ok, it works as intended, please move and rename the files as I mentioned above, resolve prettier checks and we should be done.

**Review by Tymmmy — APPROVED** [review]

**Comment by umeshSinghVerma** [comment]
Hii @Tymmmy thanks a lot, kindly attach the Hacktoberfest-accepted label to it if possible so that it might visible to Hacktoberfest also 
thanks

**Comment by umeshSinghVerma** [comment]
Thanks @Tymmmy

**Comment by devcer** [comment]
Thank you for Contributing to Interledger during Hacktoberfest. If you have any accepted PRs, You can claim your Swag [here](https://community.interledger.org/devcer/hacktoberfest-swag-redemption-claim-your-cool-gear-2152). Last date to fill up the form is December 10th, 2023. @umeshSinghVerma

**Comment by umeshSinghVerma** [comment]
thanks

**Pull request merged**